plugins/ci/skills/payload-snapshot/scripts/payload_snapshot.py

```
@@ -1361,13 +1366,24 @@ def _extract_rpmdb(self, pullspec: str, output_path: str) -> bool:
                 )
-                return cp_result.returncode == 0
+                if cp_result.returncode != 0:
+                    _log(f"    podman cp failed (exit {cp_result.returncode}) "
+                         f"for {pullspec}")
+                    stderr = cp_result.stderr.strip()
+                    if stderr:
+                        _log(f"    stderr: {stderr}")
+                    return False
+                return True
             finally:
                 subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)